buildspec.yml

```diff
@@ -1,29 +1,62 @@
 version: 0.2
+run-as: root
 
 phases:
+
+  install:
+    commands:
+      - echo Setting up required prerequisites
+      - pip install --upgrade awscli
+      - curl --silent --location -o /usr/local/bin/kubectl "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
+      - chmod +x /usr/local/bin/kubectl
+      - curl --silent --location -o /usr/local/bin/aws-iam-authenticator "https://amazon-eks.s3-us-west-2.amazonaws.com/1.14.6/2019-08-22/bin/linux/amd64/aws-iam-authenticator"
+      - chmod +x /usr/local/bin/aws-iam-authenticator
+      - curl --silent --location "https://github.com/weaveworks/eksctl/releases/download/v0.72.0/eksctl_Linux_amd64.tar.gz" | tar xz -C /tmp
+      - mv -v /tmp/eksctl /usr/local/bin
+      - chmod +x /usr/local/bin/eksctl
+      - AWS_ACCOUNT_ID=684910554524
```

Comment:
Any way to not hardcode the account id?

Reply:
Yes, we will move these files to Cloud Formation and the account id will be dynamic then.